EIPS/eip-7762.md

```diff
@@ -13,56 +13,98 @@ requires: 4844
 
 ## Abstract
 
-This EIP proposes an increase to the MIN_BASE_FEE_PER_BLOB_GAS to speed up price discovery on blob space. 
+This EIP proposes increasing `MIN_BASE_FEE_PER_BLOB_GAS`, automating blob base fee updates using target blob count, and normalizing excess gas to prevent abrupt base fee changes during forks.
 
 ## Motivation
 
-When scoping 4844, the thinking was that blobs would only enter price discovery once, relatively quickly after the blob rollout; however, this has not been the case. In fact, blobs have entered price discovery several times, and the frequency of price discovery events is likely to increase in the short term as we approach saturation of capacity. Moreover, the roadmap calls for further increases in blob capacity in subsequent hardforks, which may lead to price discovery events happening around those changes in the future. 
+With the experience of several months running EIP-4844, we have identified several minor issues with the blob base fee mechanism that we propose to address. We attempt to do so in a single EIP, summarizing the changes for simplicity.
 
-Increasing the MIN_BASE_FEE_PER_BLOB_GAS will speed up price discovery on blob space.
+First, increasing the MIN_BASE_FEE_PER_BLOB_GAS will speed up price discovery on blob space. Second, automating the blob base fee update fraction by using the target blob count in its calculation, we avoid having to update it in every future hard fork that touches the blob target. Third, normalizing excess gas will prevent blob base fee drops at forks that increase the target blob number. Eventually, symmetricising the blob base fee mechanism is discussed but not yet proposed.
 
 ## Specification
 
-The only specification change introduced by this EIP is setting MIN_BASE_FEE_PER_BLOB_GAS to 2**25:
+The specification change introduced by the first part of this EIP is setting MIN_BASE_FEE_PER_BLOB_GAS to 2**25:
 
 ```diff
 + MIN_BASE_FEE_PER_BLOB_GAS  =  2**25
 - MIN_BASE_FEE_PER_BLOB_GAS = 1
 ```
 
-## Rationale
-
-The current MIN_BASE_FEE_PER_BLOB_GAS is 1 wei. This is many orders of magnitude lower than the prevailing price of blobs when blobs enter price discovery. Whenever demand for blobs exceeds supply, blobs enter price discovery, but traversing the 8 orders of magnitude between 1 wei and the point where elasticity of demand starts to decrease takes a long time.
+Furthermore, to avoid sudden changes in the blob base fee, the excess gas MUST be reset to 0 at the time of the fork.
```

Comment:
This part alone requires a lot of testing: The t8n interface does not support a way of us telling "this is the fork block", so to work around it, we need to send hard-coded values in EEST at fork time, which requires framework changes.